crates/task-impls/src/da.rs

```diff
@@ -196,6 +206,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                 }
 
                 let payload_commitment = vid_commitment(proposal.data.encoded_transactions.clone());
```

Comment:
We don't want to clone the whole payload just to make a VID commitment. I'm surprised clippy doesn't complain.